### PR TITLE
SOC-1902 Update Discussion Email controllers

### PR DESCRIPTION
**Pull request by jsutterfield**
Ticket: https://wikia-inc.atlassian.net/browse/SOC-1902

This PR makes some small updates to the discussion email controllers in preparation for sending email notifications for the discussion service. The changes mostly involve renaming fields to align with the payload from the notification service.

cc// @mleczey, @slayful

### CODE REVIEW

File: extensions/wikia/Email/Controller/DiscussionController.class.php [hunk 1]
```diff
@@ -48,20 +52,22 @@ public function body() {
...
+    protected function getDetails() {
+        return $this->postContent;
+    }
 
     private function getButtonText() {
         return $this->getMessage( 'emailext-discussion-button-text' )->text();
     }
 
-    protected function getFooterMessages() {
+    protected function getContentFooterMessages() {
```
Comment (jsutterfield) [inline extensions/wikia/Email/Controller/DiscussionController.class.php:70]: There are 2 similar named methods used to populate footer messages in emails. The first, `getFooterMessages`, is used to populate the messages at the far bottom of the email. The second, `getContentFooterMessages`, is used to populate the last messages in the main body of the email. I accidentally had those switched, so I renamed this to use the right method name.

File: extensions/wikia/Email/Controller/DiscussionController.class.php [hunk 2]
```diff
@@ -160,8 +154,8 @@ protected static function getEmailSpecificFormFields() {
 
 class DiscussionUpvoteController extends DiscussionController {
 
-    private $firstPostContent;
-    private $upvoteCount;
+    protected $postTitle;
```
Comment (slayful) [inline extensions/wikia/Email/Controller/DiscussionController.class.php]: Does the `$postTitle` need to be `protected`?

**Comment by slayful** [comment]
Besides my question related to the `protected` field - LGTM!

File: extensions/wikia/Email/Controller/DiscussionController.class.php (hunk 2 above)
Comment (jsutterfield) [inline extensions/wikia/Email/Controller/DiscussionController.class.php]: Nope, it certainly doesn't. I'l update.

**Pull request merged**